modulesync.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'gitlab', '~>4.0'
   spec.add_runtime_dependency 'octokit', '>=4', '<9'
   spec.add_runtime_dependency 'puppet-blacksmith', '>= 3.0', '< 8'
-  spec.add_runtime_dependency 'thor'
+  spec.add_runtime_dependency 'thor', '1.3.0'
```

Comment:
Should it have pinned to NOT 1.3.0?

Reply:
I think that's correct? 1.3.1 has a regression and https://github.com/rails/thor/pull/878 contains a potential fix.

Reply:
I meant to pin to `!= 1.3.1`. I shouldn't send comments as I'm changing trains using the mobile app. Never ends well

Reply:
ah ok. I considered that, but I don't know if the fix will be in 1.3.2 or later. So I think the safer way is pinning to 1.3.0 and dependabot will inform us for new releases.

Reply:
The reason I mention it is because dependabot also informed us about 1.3.1, but yes, this stuff is always annoying